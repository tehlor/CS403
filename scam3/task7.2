(define x 0)
(sdisplay 5 (mystery x)) (println)
(sdisplay 5 (ps-mystery x)) (println)
(sdisplay 5 (acc-mystery x)) (println)
(sdisplay 5 (super-mystery x)) (println)
(symbolic-mystery)
