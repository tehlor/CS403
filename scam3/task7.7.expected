(1.0000000000,-2.0000000000,0.6666666667,-0.0888888889,0.0063492063,...)
(1.0000000000,-1.0000000000,-0.3333333333,-0.4222222222,-0.4158730159,...)
(-0.5000000000,-0.4117647059,-0.4162962963,-0.4161431949,-0.4161469032,...)
(1.0000000000,-0.5000000000,-0.4160749316,-0.4161468411,-0.4161468365,...)
(mystery x) is $\cos x$
